gensim/models/word2vec.py

```diff
@@ -918,6 +978,18 @@ def save(self, *args, **kwargs):
         super(Word2Vec, self).save(*args, **kwargs)
     save.__doc__ = utils.SaveLoad.save.__doc__
 
+class Sentences(object):
+    def __init__(self, filename):
+        self.filename = filename
+
+    def __iter__(self):
+        for line in utils.smart_open(self.filename):
+            line = utils.to_unicode(line)
+            line = line.strip()
+            words = [token.lower() for token in line.split(" ")]
```

Comment:
The only difference is the removal of newlines etc before and after a sentence. Also, it expects one sentence per line